app/Services/ImportService.php

```diff
@@ -161,8 +161,8 @@ private function createFeed(array $feed_elt, FreshRSS_Category $category, bool $
 				case strtolower(FreshRSS_Export_Service::TYPE_XML_XPATH):
 					$feed->_kind(FreshRSS_Feed::KIND_XML_XPATH);
 					break;
-				case strtolower(FreshRSS_Export_Service::TYPE_JSON_DOTPATH):
-					$feed->_kind(FreshRSS_Feed::KIND_JSON_DOTPATH);
+				case strtolower(FreshRSS_Export_Service::TYPE_JSON_DOTNOTATION):
```

Comment:
It might be sufficient to add another case line here to support OPML exports with the previous name

Reply:
Done in https://github.com/FreshRSS/FreshRSS/pull/6369/commits/cf17221f778f80c653b255798624fdb550e95b0d
Test welcome

Reply:
I am not so a big fan of having legacy code because of changed feature within a dev release, but it is ok.

Reply:
Tested to work.
`edge` is our rolling release, and there are likely many users of the JSON feature already, so avoiding breaking changes is always good. I have myself this feature deployed on multiple instances already.